lib/commands/exam.js

```diff
@@ -358,6 +358,35 @@ module.exports = TestCommand.extend({
       } else {
         this.socket.emit('get-more-tests-response', moduleName);
       }
+
+      const browserId = (() => {
+        const testPage = this.launcher.settings.test_page;
+        return /_browser=\s*(.*)/.exec(testPage)[1];
+      })();
+
+      proto.moduleMap[browserId] = proto.moduleMap[browserId] || [];
+      proto.moduleMap[browserId].push(moduleName);
+    }
+
+    events['after-tests-complete'] = function () {
+      const proto = Object.getPrototypeOf(this);
+      const fileName = 'test-execution-%TIMESTAMP%.json'.replace('%TIMESTAMP%', Date.now());
+
+      if(!proto.finishedBrowser) {
+        proto.finishedBrowser = 1;
+      }
+
+      const testPages = config.testPage;
+      const browserCount = Object.keys(testPages).length;
+
+      if (proto.finishedBrowser == browserCount) {
+        const testExecutionJson = JSON.stringify(proto.moduleMap);
+        fs.writeFile(fileName, testExecutionJson, (err) => {
+          if (err) throw err;
+        });
+      }
+
+      proto.finishedBrowser += 1;
```

Comment:
My comment above applies - I would use 'proto.finishedBrowsers' (with an 's') or proto.numFinishedBrowsers to be even more clear. 'finishedBrowser' is definitely not clear.